plugins/nginx-vhosts/commands

```diff
@@ -49,6 +49,14 @@ case "$1" in
       shopt -u nullglob
     fi
 
+    DOKKU_APP_CIDS=$(get_container_ids $APP)
+    if [[ -n $DOKKU_APP_CIDS ]]; then
```

Comment:
Hrm. I may have led you astray. Might be best to spin up a new container with the APP image and cp from just one container. Alternatively, maybe just `docker cp` from the first CID in the list.